Pod/Classes/NSManagedObject+VOKManagedObjectAdditions.h

```diff
@@ -148,10 +148,10 @@ typedef void(^VOKManagedObjectsReturnBlock)(VOKArrayOfManagedObjects *managedObj
                                forManagedObjectContext:(nullable NSManagedObjectContext *)contextOrNil;
 
 /*
- Returns one entite matching the predicate. Asserts the count is exactly 1. If more objects are returned this method will let you know.
+ Returns one entity matching the predicate. Asserts the count is exactly 1. If more objects are returned, the last one returned by Core Data will be returned.
```

Comment:
how about adding that the "If more objects are returned..." only applies if assertions are turned off?